src/components/location/location.service.ts

```diff
@@ -222,11 +222,16 @@ export class LocationService {
   async update(input: UpdateLocation, session: ISession): Promise<Location> {
     const location = await this.readOne(input.id, session);
 
+    const inputNormalized = {
+      ...input,
+      iso31663: input.iso31663?.toUpperCase(),
```

Comment:
Would you mind putting this as an a transform decorator? Sorry to be mum on that previously.

```tsx
@Transform(lodash.toUpper)
```

Reply:
No prob, I don't know much about decorators so good to know